tests/publiccloud/prepare_tools.pm

```diff
@@ -47,7 +47,7 @@ sub run {
         assert_script_run("pip3 install -q keyring");
     }
     elsif (is_sle) {
-        zypper_call('-q in aws-cli');
+        zypper_call('-q in --force-resolution aws-cli');
```

Comment:
Why with `--force-resolution`? What kind if issue is there?

Reply:
I found a dependency problem which is new and didn't happened before: http://fromm.arch.suse.de/tests/6530/file/serial_terminal.txt

But I didn't want to dig into it because we will change this approach soon and will create the HDD with autoyast.

Reply:
Problem is in usage of python development repositories in obs. Repository devel:/languages:/python:/backports/SLE_15 have python3-PyYAML 5.x but aws-cli requires version < 4.0.

Reply:
@czerw what would you recommend in such case?

Reply:
Would be good to get rid of devel languages python repo together with pip, but it is bigger task for next sprint. So, if it work now, but we need to fix it. I can have look on it in next sprint.